scenarios/evaluate-app-endpoint/data.jsonl

```diff
@@ -1,3 +1,3 @@
-{"question":"When was United Stated found ?", "ground_truth":"1776"}
-{"question":"What is the capital of France?", "ground_truth":"Paris"}
-{"question":"Who is the best tennis player of all time ?", "ground_truth":"Roger Federer"}
\ No newline at end of file
+{"query":"When was United Stated found ?", "response":"1776", "question":"When was United Stated found ?"}
```

Comment:
Do we need question for any evaluator anymore ?

Reply:
No

Reply:
Let's remove question and use query only.